connector/producer.py

```diff
@@ -45,7 +46,7 @@
 @openerp.api.returns('self', lambda value: value.id)
 def create(self, vals):
     record_id = create_original(self, vals)
-    if self.pool.get('connector.installed') is not None:
+    if is_module_installed(self.env, 'connector') is not None:
```

Comment:
'if not is_module_installed()' is preferred.

Curious to see you passing self.env instead of self. Am I missing something?